flycheck.el

```diff
@@ -6918,8 +6919,40 @@ See URL `http://www.erlang.org/'."
   :error-patterns
   ((warning line-start (file-name) ":" line ": Warning:" (message) line-end)
    (error line-start (file-name) ":" line ": " (message) line-end))
+  :enabled (lambda () (not (flycheck-rebar-config-exists-p)))
```

Comment:
Please remove this `:enabled`… it prevents users from explicitly selecting the plain `erlang` checker in rebar projects even if they want to.  Just move `erlang-rebar3` before `erlang` and Flycheck will automatically prefer the rebar checker.